tests/unit/services/test_policy_service_unit.py

```diff
@@ -0,0 +1,72 @@
+import pytest
+import json
+from unittest.mock import MagicMock
+
+
+from policyengine_api.services.policy_service import PolicyService
+
+from tests.fixtures.policy_fixtures import (
+    valid_policy_format,
+    # valid_hash_value,
+    # mock_hash_object,
+    existing_policy_record,
+)
+
+service = PolicyService()
+
+
+class TestGetPolicyJson:
+    def test_get_policy_json_given_existing_record(
+        self, test_db, existing_policy_record
+    ):
+
+        # GIVEN an existing record... (included as fixture)
+
+        # WHEN we call get_policy_json for this record...
+        result = service.get_policy_json(
+            valid_policy_format["country_id"], valid_policy_format["id"]
+        )
+
+        valid_policy_json = valid_policy_format["policy_json"]
+
+        # THEN result should be the expected policy json
+        assert result == valid_policy_json
+
+    def test_get_policy_json_given_nonexisting_record(self, test_db):
+
+        # GIVEN an empty database... (created by default)
+
+        # WHEN we call get_policy_json for nonexistent record...
+        NO_SUCH_RECORD_ID = 999
+        result = service.get_policy_json("us", NO_SUCH_RECORD_ID)
+
+        # THEN result should be the expected policy json
```

Comment:
__nit:__ I believe this comment is incorrect
```suggestion
        # THEN result should be the expected policy json
```